packages/graphql/src/post.ts

```diff
@@ -198,3 +198,33 @@ export const UndoBookmarkPostMutation = graphql(
   }`,
 );
 export type UndoBookmarkPostRequest = RequestOf<typeof UndoBookmarkPostMutation>;
+
+const DeletePostResult = graphql(
+  `fragment DeletePostResult on DeletePostResult {
+    ... on DeletePostResponse {
+      __typename
+      hash
+    }
```

Comment:
Can you please define a `DeletePostResponse` fragment, export it, and use it here?